core/app/models/spree/order.rb

```diff
@@ -27,6 +27,25 @@ def generate_number(options = {})
     alias :display_ship_total :display_shipment_total
     alias_attribute :ship_total, :shipment_total
 
+    MONEY_THRESHOLD  = 999_999
```

Comment:
Why only 999_999? Shouldn't this be as large as databases allow the 2 billion something number I think we added somewhere else, but not seeing it at the moment... Thought we had set it on line item total or something perhaps it's still an open PR